### ISSUE TITLE
[QUESTION] Tool tip for precipitation forecast from MOSMIX data source

### ISSUE THREAD

**Issue by Dondo11** [Daveiano/weewx-wdc#287]
Hi Daveiano

I'm using weewx-wdc to visualise forecast data obtained from DWD via @roe-dl's weewx-DWD.

When hovering over the precipitation graph, the tool tip always shows the same time of day for any point on the x-axis.  According to @roe-dl, this may be due to the particular mix, in this graph, of hourly data for both, cloud cover and precipitation probabilty vs. daily totals for precipitation itself.

I realise that showing the time of day provided by cloud cover/probabilty may be similarly puzzling in conjunction with daliy precipitation totals yet more informative in general.  

Can you think of a way to change the current behaviour?

![TT 1](https://github.com/user-attachments/assets/84f759a9-89a9-4454-a66a-b63e627d6a39)
![TT 2](https://github.com/user-attachments/assets/de513816-d2f5-4771-9eae-41f16119d1f3)
![TT 3](https://github.com/user-attachments/assets/4e3a4459-814d-4581-85c8-bc7538f47b29)

**Comment by Dondo11** [comment]
Closing due to comment on issue @roe-dl's project.  Thanks!  Please file an issue if it's now trouble, since I'm not a developer myself.

**Comment by Dondo11** [comment]
Just been experimenting with skin.conf a bit, and moving the "rain" section in [[[[dwd_meteogram_2]]]] from being first is a workaround.

![TT 1-1](https://github.com/user-attachments/assets/082df35f-5964-4cfa-b690-1c6c506fbaf3)
<img width="705" alt="TT 2-2" src="https://github.com/user-attachments/assets/00dafcb8-dcd9-44f0-a1a7-8e319973d958">

**Comment by Daveiano** [comment]
> Just been experimenting with skin.conf a bit, and moving the "rain" section in [[[[dwd_meteogram_2]]]] from being first is a workaround.

Good catch! I had the same workaround in mind, but I had no time yet to test it out. 

Does this fully work and solve the problem? If so, I would add this workaround in the FAQ and Wiki of the project.

**Comment by Dondo11** [comment]
> Does this fully work and solve the problem? If so, I would add this workaround in the FAQ and Wiki of the project.

As far as I can tell it works perfectly well.  The time/date values change according to the position on x-axis for precipitation probability and cloud cover while the value for total rainfall/day remains constant.  So this workaround is viable as long as the order of values is irrelevant.
